From randy.dunlap@oracle.com  Mon Mar 15 15:06:08 2010
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Wed, 10 Mar 2010 11:47:58 -0800
Subject: driver-core: fix missing kernel-doc in firmware_class
Cc: gregkh@suse.de
Message-ID: <20100310114758.622bfff2.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix kernel-doc warning in firmware_class.c:

Warning(drivers/base/firmware_class.c:94): No description found for parameter 'attr'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/firmware_class.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/base/firmware_class.c
+++ b/drivers/base/firmware_class.c
@@ -78,6 +78,7 @@ firmware_timeout_show(struct class *clas
 /**
  * firmware_timeout_store - set number of seconds to wait for firmware
  * @class: device class pointer
+ * @attr: device attribute pointer
  * @buf: buffer to scan for timeout value
  * @count: number of bytes in @buf
  *
